### DISCUSSION TITLE
SVN repository import failed

### DISCUSSION THREAD

**Discussion by apirozzi** [community/community#69915] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,
I'm a new github user.
I tried to import my svn project, from my repository to github, with the site feature.
After some hour it returns:
```
We found an error during import
There was an error importing commits
We recommend restarting the import process, but if you are stille having trouble contact support
```
![Immagine WhatsApp 2023-10-12 ore 16 01 24_59f719b9](https://github.com/community/community/assets/147624215/8bcd1791-2ce7-4c4d-a775-53f1fcfb47d1)

Naturally I tried many times...

How can I solve this?

**Comment by bio** [comment]
GitHub officially no longer supports SVN.
https://github.blog/changelog/2024-01-08-subversion-has-been-sunset/
